### DISCUSSION TITLE
Welcome to mgpusim Discussions!

### DISCUSSION THREAD

**Discussion by syifan** [sarchlab/mgpusim#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yzgll** [comment]
Why does it prompt a bad match when I run AES on mgpusim no matter how much I modify the length, the CPU result is inconsistent with the GPU result, and only when the length is the default 65536 it works correctly?

**Reply by syifan** [reply]
The question is answered [here](https://github.com/sarchlab/mgpusim/discussions/54).
